workmanager/android/src/main/kotlin/dev/fluttercommunity/workmanager/Extractor.kt

```diff
@@ -213,13 +255,14 @@ object Extractor {
                     constraintsConfig = extractConstraintConfigFromCall(call),
                     outOfQuotaPolicy = extractOutOfQuotaPolicyFromCall(call),
                     backoffPolicyConfig =
-                        extractBackoffPolicyConfigFromCall(
```

Comment:
@ened is this formatting ok, or it should be reverted. To not add unncessary changed lines and simplify reviews single formatting should be used